apps/studio/src/pages/signUp/SignUp.tsx

```diff
@@ -18,18 +18,24 @@ interface AccountValues {
   readonly confirmPassword: string;
   readonly email: string;
   readonly newPassword: string;
+  readonly referrer: string;
 }
 
 const SignUp: FunctionComponent = () => {
   const theme = useTheme();
   const dispatch = useAppDispatch();
   const currentPathLocation = useLocation();
 
+  // Parse query parameters for referral code
+  const queryParams = new URLSearchParams(currentPathLocation.search);
+  const referrerCode = queryParams.get("mwr") || "";
```

Comment:
Should this also hide behind the LD flag, I don't believe it will do anything at this point without the Studio production campaign is created.